qiskit/providers/models/backendconfiguration.py

```diff
@@ -610,3 +686,43 @@ def describe(self, channel: ControlChannel) -> Dict[DriveChannel, complex]:
         for u_chan_lo in self.u_channel_lo[channel.index]:
             result[DriveChannel(u_chan_lo.q)] = u_chan_lo.scale
         return result
+
+    def _parse_channels(self, channels: Dict[set, Any]) -> Dict[Any, Any]:
+        r"""
+        Generates a dictionaries of ``Channel``\s, and tuple of qubit(s) they operate on.
+
+        Args:
+            channels: An optional dictionary containing information of each channel -- their
+                purpose, type, and qubits operated on.
+
+        Returns:
+            qubit_channel_map: Dictionary mapping tuple of qubit(s) to list of ``Channel``\s.
+            channel_qubit_map: Dictionary mapping ``Channel`` to list of qubit(s).
+            control_channels: Dictionary mapping tuple of qubit(s), to list of
+                ``ControlChannel``\s.
+        """
+        qubit_channel_map = defaultdict(list)
+        channel_qubit_map = defaultdict(list)
+        control_channels = defaultdict(list)
+        example = {
```

Comment:
is this a good name?

Reply:
How about `channels_dict`?

Reply:
Fixed in [e3e4649](https://github.com/Qiskit/qiskit-terra/pull/4097/commits/e3e4649ebde3e01418ea34d4d619b231446a2970)